components/CalendarPrint/CalendarStyleSelectionCard.tsx

```
@@ -0,0 +1,51 @@
+            {calendarPrintStyle && <calendarPrintStyle.component {...data} />}
+          </div>
+        </div>
+        <div className="flex flex-row flex-wrap gap-2 justify-center">
+        {futureYears.map((year) => (
+          <Button
+          variant="default"
+          className="flex-1"
+          onClick={() => {
+            window.open(`/calendar-prints/${calendarId}?year=${year}`, '_blank')
+          }}
+        >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)